core/extension/gdnative_interface.cpp

```diff
@@ -766,8 +766,20 @@ static GDNativeTypePtr gdnative_packed_vector3_array_operator_index(GDNativeType
 }
 
 static GDNativeTypePtr gdnative_packed_vector3_array_operator_index_const(const GDNativeTypePtr p_self, GDNativeInt p_index) {
+	PackedVector3Array *self = (PackedVector3Array *)p_self;
+	ERR_FAIL_INDEX_V(p_index, self->size(), nullptr);
+	return (GDNativeTypePtr)&self[p_index];
+}
+
+static GDNativeVariantPtr gdnative_array_operator_index(GDNativeTypePtr p_self, GDNativeInt p_index) {
```

Comment:
Names and implementations don't seem to match? I think you got confused in the copy-paste.

Reply:
You modified `gdnative_packed_vector3_array_operator_index_const` to lose `const`ness, so it's now the same as `gdnative_packed_vector3_array_operator_index`.

`gdnative_array_operator_index` is what `gdnative_packed_vector3_array_operator_index_const` should be.
And `gdnative_operator_index_const` lacks `array` in the name.

Reply:
yeah all these functions are just looking too much alike, I must have been crossed eyed or something. 
Hopefully its good now :)